android-project/app/src/main/AndroidManifest.xml

```diff
@@ -35,6 +35,9 @@
     <uses-permission android:name="android.permission.BLUETOOTH" />
     <!-- Allow access to the vibrator -->
     <uses-permission android:name="android.permission.VIBRATE" />
+    <!-- Allow access to the network for multiplayer -->
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Comment:
Since we do not make use of this API i think it's premature to request access to it. I'm not sure it would be relevant either since it is not the only way for a device to join a game.

Reply:
True, that was more for completeness.